sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceClientBuilderTest.java

```diff
@@ -220,6 +220,16 @@ public void singleFormOfAuthenticationPresent() {
             .buildAsyncClient());
     }
 
+    // Tests if the endpoint is malformed, like in the case it includes the tableName
+    @Test
+    public void malformedEndpointThrowsIllegalArgumentException() {
+        assertThrows(IllegalArgumentException.class, () -> new TableServiceClientBuilder()
+            .credential(new AzureNamedKeyCredential("name", "key"))
+            .endpoint("https://myAccount.table.core.windows.net/tableName")
+            .buildAsyncClient()
+        );
+    }
+
```

Comment:
It looks like this test was left over from the previous approach. We should delete it as the builders don't check URLs for this.

Reply:
Thank you, will do.